packages/babel-plugin-transform-bigint/test/fixtures/addition/expected.js

```diff
@@ -0,0 +1,2 @@
+import _BigInt from "bigint-polyfill";
```

Comment:
would be nice if we could use in the REPL but I guess that's not possible in this case (or we have to fake it)